unit_tests/ngp_algorithms/UnitTestSSTMaxLengthScaleAlg.C

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+
+#include "kernels/UnitTestKernelUtils.h"
+#include "UnitTestHelperObjects.h"
+
+#include "AlgTraits.h"
+#include "ngp_algorithms/SSTMaxLengthScaleAlg.h"
+#include "ngp_algorithms/SSTMaxLengthScaleDriver.h"
+#include "utils/StkHelpers.h"
+
+TEST_F(SSTKernelHex8Mesh, NGP_SST_Max_Length_Scale)
+{
+  // Only execute for 1 processor runs
+  if (bulk_.parallel_size() > 1) return;
+
+  fill_mesh_and_init_fields();
+
+  // zero out fields
+  stk::mesh::field_fill(0.0, *maxLengthScale_);
+
+  unit_test_utils::HelperObjects helperObjs(
+    bulk_, stk::topology::HEX_8, 1, partVec_[0]);
+
+  // Force computation of edge area vector
+  helperObjs.realm.realmUsesEdges_ = true;
+
+  const auto& fieldMgr = helperObjs.realm.mesh_info().ngp_field_manager();
+  auto& ngpElemVol = fieldMgr.get_field<double>(
+      elementVolume_->mesh_meta_data_ordinal());
+
```

Comment:
Looks like this is leftover from cut & paste from another test, and is not needed?